core/src/messages/cert.rs

```diff
@@ -20,12 +20,12 @@ impl TypedMessage for SetAcmeDnsRecord {
     }
 
     fn tmp_alt_subject(&self) -> Option<String> {
-        Some(format!("cluster.{}.set_acme_record", self.cluster))
```

Comment:
this will break backwards compatibility, but we are intentionally breaking backwards compatibility soon anyway and this particular instance is low-risk

Reply:
Will this cause https downtime when we switch over?

Reply:
No, drones that have already obtained certs will continue to be able to use them. As long as we roll the drones before their certs expire, we'll avoid any downtime.